custom_components/tuya_local/devices/ferroli_titano_twin.yaml

```diff
@@ -0,0 +1,64 @@
+name: Ferroli Titano Twin
+products:
+  - id: lci0ebsz7ftaruaf
+    name: DT EWH
+primary_entity:
+  entity: climate
```

Comment:
This device seems to be a water_heater rather than a climate device, and should probably use that entity instead.

Reply:
In tuya cloud the device is recognized as a climate device. i try to make the same.
![Screenshot 2023-11-19 160953](https://github.com/make-all/tuya-local/assets/88716308/819f307f-a437-4be3-b5f2-7ea7db382e2b)
Upper image  tuya cloud
Bottom image tuya-local

Reply:
The objective of this integration is not to accept the same limitations as the cloud tuya integration, but to provide the best integration of devices possible.  Water heater clearly fits this device better than climate.